lib/components/InlineMessage/index.tsx

```diff
@@ -57,7 +59,9 @@ export const InlineMessage = ({
         )}
     >
         {(variant !== "general" || icon) && (
-            <div className="deriv-inline-message__icon">
+            <div
+                className={`deriv-inline-message__icon deriv-inline-message__icon--${iconPosition}`}
```

Comment:
can we use `clsx` for this one like this:
```suggestion
                className={clsx('deriv-inline-message__icon,'  {
                'deriv-inline-message__icon--top' : top,
                'deriv-inline-message__icon--center:  center,
                'deriv-inline-message__icon--bottom: bottom
                }
```
easier to see and debug aswell if you do it like this.

Reply:
@aizad-deriv sorted